rasterio/crs.py

```diff
@@ -54,9 +57,14 @@ def from_string(prjs):
     if '{' in prjs:
         # may be json, try to decode it
         try:
-            return json.loads(prjs, strict=False)
+            val = json.loads(prjs, strict=False)
         except ValueError:
-            raise ValueError('crs appears to be JSON but is not valid')
+            raise CRSError('crs appears to be JSON but is not valid')
```

Comment:
@perrygeo Switched to `CRSError`, but will happily revert if anyone changes their mind.